lib/src/main/scala/code/chymyst/jc/ReactionSite.scala

```diff
@@ -428,5 +445,4 @@ private[jc] final class ExceptionEmittingSingleton(message: String) extends Exce
 private[jc] final class ExceptionNoReactionPool(message: String) extends ExceptionInJoinRun(message)
 private[jc] final class ExceptionNoWrapper(message: String) extends ExceptionInJoinRun(message)
 private[jc] final class ExceptionWrongInputs(message: String) extends ExceptionInJoinRun(message)
-private[jc] final class ExceptionEmptyReply(message: String) extends ExceptionInJoinRun(message)
```

Comment:
Yes, your PR was getting rid of the same exception. I built upon the main idea of your PR.